app/javascript/crayons/guidelines/__stories__/Utility-first-css.stories.mdx

```diff
@@ -22,12 +22,12 @@ like this:
 This element has several classes applied. Each responsible for different part of
 styling:
 
-|**Utility Class**|True CSS|
-|---|---|
-|**`radius-default`**|`border-radius: 5px;`|
-|**`bg-accent-brand`**|`background-color: #3b49df;`|
-|**`p-6`**|`padding: 24px;`|
-|**`mb-4`**|`margin-bottom: 16px;`|
+| **Utility Class**     | True CSS                     |
```

Comment:
These style changes are getting automatically applied in my editor whenever I save the file. Kept these as this makes this file more readable + the output is not affected.